packages/fluentui/react-northstar/src/components/Dropdown/Dropdown.tsx

```diff
@@ -51,6 +52,13 @@ import {
   partitionPopperPropsFromShorthand,
 } from '../../utils/positioner';
 
+export type unhandledA11yProps = {
+  props: {
+    'aria-invalid': string;
+    'aria-labelledby': string;
```

Comment:
Please is that correct that we are extracting `aria-labelledby`, but not using it anywhere below?
Like in https://github.com/microsoft/fluentui/pull/16324/files#diff-0dfcfdca092d43b516fbc6007112238d042697c91fdced73011fa22d664d65f2R549 where we construct final `aria-labelledby`